src/main/java/org/broadinstitute/hellbender/engine/filters/AmbiguousBaseReadFilter.java

```diff
@@ -14,23 +14,27 @@
 
     private static final long serialVersionUID = 1L;
 
-    @Argument(doc="Threshold fraction of non-regular bases (e.g. N) above which to filter",
-            fullName="ambigFilterFrac",
-            shortName="ambigFilterFrac", optional=true)
-    public float N_FRAC = 0.05f;
+    @Argument(fullName = "ambigFilterFrac", shortName = "ambigFilterFrac", optional = true)
+    public double N_FRAC = 0.05;
```

Comment:
Arguments are not constants and shouldn't be in UPPERCASE.  There are places that have them like that that haven't been refactored, but we shouldn't be adding more.